api/comment_threads.rb

```diff
@@ -1,21 +1,22 @@
 
 get "#{APIPREFIX}/threads" do # retrieve threads by course
+
   threads = CommentThread.where({"course_id" => params["course_id"]})
```

Comment:
FYI: A search for `params["..."]` (double-quote) turns up 71 matches.

@clintonb: Again, I'm not sure what you were going for with the switch to `params['...'` (single-quote) below.  There are only 22 matches for `params['...'` (single-quote), 8 of which are in this file.  We should know what our best practice is, and minimally be consistent in each file.